dotCMS/src/main/java/com/dotcms/publishing/BundlerUtil.java

```diff
@@ -213,25 +216,33 @@ public static void objectToJSON( Object obj, File f, boolean removeFirst ) {
     public static Object xmlToObject(File f){
     	XStream xstream = new XStream(new DomDriver("UTF-8"));
 
-    	BufferedInputStream input = null;
-		try {
-			input = new BufferedInputStream(Files.newInputStream(f.toPath()));
-			Object ret = xstream.fromXML(input);
-			return ret;
-		} catch (IOException e) {
-			Logger.error(BundlerUtil.class,e.getMessage(),e);
-			return null;
-		}finally{
-			try {
-				input.close();
-			}
-			catch(Exception e){
-
-			}
+        try (InputStream input = new BufferedInputStream(Files.newInputStream(f.toPath()))) {
+            return xstream.fromXML(input);
+		} catch (Exception e) {
+			Logger.warnAndDebug(BundlerUtil.class,e.getMessage(),e);
+			return xmlToObjectWithPrologue(f);
 		}
-
 	}
 
+    /**
+     * Adds a XML 1.1 Prologue before trying to deserialize
+     * @param file
+     * @return
+     */
+    private static Object xmlToObjectWithPrologue(File file) {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Parameter 'file' is not assigned and could be declared final](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=5632525)